src/os/ui/ogc/ifeaturetypedescriptor.js

```diff
@@ -0,0 +1,30 @@
+goog.provide('os.ui.ogc.IFeatureTypeDescriptor');
+
+
+
+/**
+ * Interface for OGC feature type.
+ * @interface
+ */
+os.ui.ogc.IFeatureTypeDescriptor = function() {};
+
+
+/**
+ * @type {string}
+ * @const
+ */
+os.ui.ogc.IFeatureTypeDescriptor.ID = 'os.ui.ogc.IFeatureTypeDescriptor';
+
+
+/**
+ * Get the WFS FeatureType.
```

Comment:
This isn't restricted to WFS, so you can just say "FeatureType". Arc is using this interface as well.